docs/http-api/entry-api/update_data.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Update Data
+description: HTTP Endpoints for updating data in the database.
+---
+
+<head>
+  <link
+    rel="canonical"
+    href="https://www.reduct.store/docs/http-api/entry-api/update_data"
+  />
+</head>
+
+# Entry API to Update Data
+
+The API provides several endpoints for updating labels of a record or multiple records in a single request.
+
+import SwaggerComponent from "@site/src/components/SwaggerComponent";
+
+<SwaggerComponent
+  method="PATCH"
+  path="/api/v1/b/:bucket_name/:entry_name"
+  summary="Update labels of a record"
+  description={
+    <>
+      The method allows to update labels of a existing record. It receives
+      labels in headers that start with <code>x-reduct-label-</code> and updates
+      the labels or adds new ones. If a header with a label is empty, the label
+      is removed. Due to the content of the record is immutable, the method
```

Comment:
"Because the content of the record is immutable, the method doesn't expect a body, and the content-length header is required, but must be 0."